lib/turn.rb

```diff
@@ -1,12 +1,72 @@
 class Turn
-  attr_accessor :board, :player, :turn_number
+  attr_accessor :board, :player, :computer, :turn_number
+  
   def initialize(board, player)
     @board = board
     @player = player
-    # @computer = computer
+    @computer = computer
     @turn_number = 0
+  end
+
+  def place_selection
+    p "Choose a column A-G to drop your piece!"
+    selection = gets.chomp.upcase
 
+    until @board.grid.first.include?(selection)
+      p "Please input a column letter A-G."
+      selection = gets.chomp.upcase
+    end
+    column_converter(selection)
   end
 
 
-end
+  def column_converter(selection)
```

Comment:
column converter method was added simply to show the index numbers for column A..G(0-6)